build.gradle

```diff
@@ -74,6 +75,8 @@ subprojects {
 
     checkstyle {
       configFile = file("${rootDir}/gradle/checkstyle/linkedin-checkstyle.xml")
+      configProperties = ['config_loc' : project.file("gradle/checkstyle") ]
```

Comment:
Consider using the [`configDirectory`](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.CheckstyleExtension.html#org.gradle.api.plugins.quality.CheckstyleExtension:configDirectory) property.

Also the path here should be relative to the `rootDir` and not the project dir, this should help fix the latest build failure.

Reply:
I tried using configDirectory and ran into a bunch of problems since it's a DirectoryProperty object instead of a plain file object like I expected. My preference is to stick with configFile. Hope that's ok with you.

Reply:
Sure. Should be okay, I think `dir("/path/to/dir")` should be able to get you the DirectoryProperty, but not fully sure.